pymc/distributions/continuous.py

```diff
@@ -1644,12 +1644,12 @@ class AsymmetricLaplace(Continuous):
 
     Parameters
     ----------
-    b : tensor_like of float
-        Scale parameter (b > 0).
-    kappa : tensor_like of float
-        Symmetry parameter (kappa > 0).
-    mu : tensor_like of float, default 0
-        Location parameter.
+    kappa: float
+        Symmetry parameter (kappa > 0)
+    mu: float
+        Location parameter
+    b: float
+        Scale parameter (b > 0)
```

Comment:
There have been some issues with the merge, type should still indicate `tensor_like`, the rest seems to have all updated content.